vue/components/ui/atoms/container/container.vue

```diff
@@ -104,6 +118,11 @@ body.mobile .container-ripe > .container-header {
     user-select: none;
 }
 
+.container-ripe > .container-header > .header-buttons > .header-button {
+    margin: 0px 4px 0px 0px;

```

Comment:
Shouldn't this be applied to button-colors only?

Reply:
Both `button-icon`s and `button-color`s don't have any margin between them. I guess if we wanted to add margin only to `button-color`s then yes, I need to change
`.container-ripe > .container-header > .header-buttons > .header-button {` 
to
`.container-ripe > .container-header > .header-buttons > .header-button.button-color {`

Reply:
I know they don't, but if button-icons didn't have a margin before this change, they shouldn't gain a margin now without very good reason. They don't have a margin because they only occupy the full space when they are `:hover`, so they don't need it.

Long story short: please make the change you mentioned.